extensions/resteasy-reactive/rest-client-reactive/deployment/src/test/java/io/quarkus/rest/client/reactive/registerclientheaders/RegisterClientHeadersTest.java

```diff
@@ -38,7 +38,7 @@
 
     @Test
     public void shouldUseHeadersFactory() {
```

Comment:
does it mean the previous test didn't make sense?

Reply:
It made sense before and still does. The test was basically testing headers related functionality and I only used it because it's one of the few tests that uses `@QueryParam`

Reply:
I mean the test method, you replaced a check for passing `"ping:"`  and getting `"ping:bar"` back with a test for null value, the question is if the previous check didn't make sense or did you replace it by accident and planned to add a new test method

Reply:
How about now?

Reply:
This preserves the old test, so if it still makes sense, is safer ;)